Mathlib/LinearAlgebra/ExteriorAlgebra/Basic.lean

```diff
@@ -330,10 +330,123 @@ theorem ιMulti_succ_curryLeft {n : ℕ} (m : M) :
       rfl
 #align exterior_algebra.ι_multi_succ_curry_left ExteriorAlgebra.ιMulti_succ_curryLeft
 
+variable (R)
+
+/-- The image of `ExteriorAlgebra.ιMulti R n` is contained in the `n`th exterior power. -/
+lemma ιMulti_range (n : ℕ) :
+    Set.range (ιMulti R n (M := M)) ⊆ ↑(LinearMap.range (ι R (M := M)) ^ n) := by
+  rw [Set.range_subset_iff]
+  intro v
+  rw [ιMulti_apply]
+  apply Submodule.pow_subset_pow
+  rw [Set.mem_pow]
+  exact ⟨fun i => ⟨ι R (v i), LinearMap.mem_range_self _ _⟩, rfl⟩
+
+/-- The image of `ExteriorAlgebra.ιMulti R n` spans the `n`th exterior power, as a submodule
+of the exterior algebra. -/
+lemma ιMulti_span_fixedDegree (n : ℕ) :
+    Submodule.span R (Set.range (ιMulti R n)) =
+    LinearMap.range (ι R : M →ₗ[R] ExteriorAlgebra R M) ^ n := by
+  refine le_antisymm (by rw [Submodule.span_le]; apply le_trans (ιMulti_range R n) (le_refl _)) ?_
+  rw [Submodule.pow_eq_span_pow_set, Submodule.span_le]
+  refine fun u hu ↦ Submodule.subset_span ?_
+  obtain ⟨f, hfu⟩ := Set.mem_pow.mp hu
+  rw [Set.mem_range, ← hfu]
+  existsi fun i => ιInv (f i).1
+  rw [ιMulti_apply]
+  congr
+  ext i
+  obtain ⟨v, hv⟩ := LinearMap.mem_range.mp (by simp only [SetLike.coe_mem] :
+    (f i).1 ∈ LinearMap.range (ι R (M := M)))
+  rw [← hv, ι_inj]; erw [ι_leftInverse]
+
+/-- Given a linearly ordered family `v` of vectors of `M` and a natural number `n`, produce the
+family of `n`fold exterior products of elements of `v`, seen as members of the exterior algebra. -/
+def ιMulti_family (n : ℕ) {I : Type*} [LinearOrder I] (v : I → M) :
+    {s : Finset I // Finset.card s = n} → ExteriorAlgebra R M :=
+  fun s ↦ ιMulti R n (fun i => v (Finset.orderIsoOfFin _ s.prop i))
```

Comment:
Could eliminte the `fun` here:
```suggestion
def ιMulti_family (n : ℕ) {I : Type*} [LinearOrder I] (v : I → M) (s : {s : Finset I // Finset.card s = n}) :
    ExteriorAlgebra R M :=
  ιMulti R n fun i => v (Finset.orderIsoOfFin _ s.prop i)
```
I'd be tempted to make this an `abbrev` rather than a `def`, especially since you include no theorems about it.
If it's an `abbrev`, no one needs any theorems because it because everything can see through it.